docs/src/test/scala/docs/http/scaladsl/server/directives/FormFieldDirectivesExamplesSpec.scala

```diff
@@ -13,12 +13,12 @@ class FormFieldDirectivesExamplesSpec extends RoutingSpec with CompileOnlySpec {
   "formFields" in {
     //#formFields
     val route =
-      formFields("color", "age".as[Int]) { (color, age) =>
+      formFields("color", "age".as[Int], "direction" ! "up") { (color, age, _) =>
```

Comment:
I moved it to the end. Having it at the front changed the result if the required field value was missing.